sdk/keyvault/generate.ts

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env node
```

Comment:
The end result is that this file should be deleted once all the workarounds are no longer needed